connection/Cargo.toml

```
@@ -9,6 +9,7 @@ rust-version = { workspace = true }
 
 [dependencies]
 
+der = "0.7.7"
```

As suggested by Nick in a different file
```suggestion
der = "0.7.8"
```